WcaOnRails/spec/structure_sql_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "structure.sql" do
+  let!(:structure_sql) do
+    # Load structure.sql excluding the archived phpbb3 tables as well as the Rails internal ones.
+    File.read("#{Rails.root}/db/structure.sql")
```

Comment:
Oh, interesting! I was actually expecting a solution that used ActiveRecord/the mysql gem to query what the table charsets and column charsets were, but this looks like it works. I'd like to spend a little time playing around with the rails console to see if there's some way to get to this data, so please don't merge this up yet. I'll send you a LGTM by Friday, or a proposal for a different way to test this.